src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -108,27 +108,34 @@ private void initReportSummary2HdrList(List<HearingsByHearingTypeReportSummary2H
 
     }
 
-    private String getSubSplit(HearingTypeItem hearingTypeItem) {
+    private String getSubSplitSitAlone(HearingTypeItem hearingTypeItem) {
         if ("Full".equals(hearingTypeItem.getValue().getHearingSitAlone())) {
             return "Full Panel";
-        }
-        if (YES.equals(hearingTypeItem.getValue().getHearingSitAlone())) {
+        } else if (YES.equals(hearingTypeItem.getValue().getHearingSitAlone())) {
             return "EJ Sit Alone";
+        } else {
+            return "";
         }
+    }
+
+    private String getSubSplitJM(HearingTypeItem hearingTypeItem) {
```

Comment:
The problem with this code is that it then requires a subsequent isEmpty check later on. if this method was
isSubSplitJM and returned a boolean then you could later on use
if (isSubSplitJM(...)) {
   setReportSummary2HdrFields("JM", hearingTypeItem, reportSummary2HdrList);

I think we ought to be trying to not to introduce more empty/null checks than are already embedded in the case data

Reply:
refactored